### ISSUE TITLE
[Site Request] Fuhu

### ISSUE THREAD

**Issue by Froktime** [manga-download/hakuneko#6411]
**Name of the website**
Fuhu

**Website urls (examples below)**
- Site: https://fuhuzz.com/
- Manga List: https://fuhuzz.com/the-loai/ngon-tinh-f4ee59ed_fh-sentimental.html
- Manga example: https://fuhuzz.com/comic/em-trai-chong-say-dam-toi-_eIBoew0J.html
- Chapter online viewer example: https://fuhuzz.com/comic-chapter/em-trai-chong-say-dam-toi-0-mo-dau_Nsx9cw9D.html
https://api.funhub.net/fhredirect/QTq5NG75SOZLrQK8M0nyD8oHCrbqEAA5uPi7LiSncqI

**Languages**
Vietnamese

**Website relationship*
Haven't

**Additional details**
This website embeds reading link. The actual reading link is api.funhub. com.....

**Comment by MikeZeDev** [comment]
> Manga List: https://fuhuzz.com/the-loai/ngon-tinh-f4ee59ed_fh-sentimental.html

Is that a manga list for a single genre? Dont they have a full manga list? Sorry i dont read vietnamese you have to help me.

**Comment by Froktime** [comment]
> > Manga List: https://fuhuzz.com/the-loai/ngon-tinh-f4ee59ed_fh-sentimental.html
> 
> Is that a manga list for a single genre? Dont they have a full manga list? Sorry i dont read vietnamese you have to help me.

Yes. This site contains many genre (Comic, Novel, Movie), so it doesn't have a specific listing page

**Comment by MikeZeDev** [comment]
Its not related. Most websites have a single list along with genre-filtered lists.  Having to parse many lists is annoying :/

**Comment by MikeZeDev** [comment]
dead
